app/main/services/search_service.py

```diff
@@ -1,16 +1,19 @@
-from flask import current_app, url_for
 from elasticsearch import TransportError
+from flask import current_app, url_for
+
+from dmutils.timing import logged_duration_for_external_request
 
-from ... import elasticsearch_client as es
 import app.mapping
-from app.main.services.response_formatters import \
-    convert_es_status, convert_es_results, generate_pagination_links
+from app.main.services.response_formatters import convert_es_status, convert_es_results, generate_pagination_links
 from app.main.services.query_builder import construct_query
 
+from ... import elasticsearch_client as es
+
 
 def refresh(index_name):
     try:
-        es.indices.refresh(index_name)
+        with logged_duration_for_external_request('es'):
```

Comment:
So I guess the idea is we'll be able to figure out what "type" of es request it is by looking at the line number and digging rather than setting the "description" for each...

Reply:
🤔 🤷‍♂️ Dunno. Just chucked it on there as it was used elsewhere. I think if we need descriptions we can add them later?

Reply:
I think it's fine - the function name is included? https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/timing.py#L141

Reply:
(the function name of the *parent* is included, which is ambiguous if that function makes more than one request)